netlogo-gui/src/main/lab/Worker.scala

```diff
@@ -118,6 +118,10 @@ class Worker(val protocol: LabProtocol)
           else if(name.equalsIgnoreCase("RANDOM-SEED"))
             ws.world.mainRNG.setSeed(value.asInstanceOf[java.lang.Double].longValue)
         }
+        if(d.minPxcor >= d.maxPxcor || d.minPycor >= d.maxPycor)
+          throw new FailedException("Minimum x and y coordinates should be greater than the maximum coordinates")
```

Comment:
This error text is just slightly off, should be either of the following:
* "Minimum x and y coordinates should be less than the maximum coordinates"
* "Maximum x and y coordinates should be greater than the minimum coordinates"